README.md

```diff
@@ -107,84 +126,51 @@ topic     = 'user_created'
 channel   = 'my_consuming_service'
 processor = MessageProcessor
 
-FastlyNsq::Listener.new(topic: topic, channel: channel, processor: processor).go(run_once: true)
+FastlyNsq::Listener.new(topic: topic, channel: channel, processor: processor)
 ```
 
-This will pop the next message
+This will send messages through `FastlyNsq.manager.pool`
 off of the queue
 and send the JSON text body
-to `MessageProcessor.process(message_body, topic)`.
+to `MessageProcessor.call(message)`.
 
-To initiate a blocking loop to process messages continuously:
+Specify a topic priority by providing a number (default is 5)
```

Comment:
DEFAULT_PRIORITY is set to 0, unless I'm missing something.

Reply:
Nope, you are right.  I read `DEFAULT_CONNECTION_TIMEOUT` instead.